cartography/intel/aws/cloudwatch.py

```diff
@@ -0,0 +1,195 @@
+import time
+import logging
+from typing import Dict
+from typing import List
+
+import boto3
+import neo4j
+
+from cartography.util import aws_handle_regions
+from cartography.util import run_cleanup_job
+from cartography.util import timeit
+
+logger = logging.getLogger(__name__)
+
+
+@timeit
+@aws_handle_regions
+def get_cloudwatch_alarm(boto3_session: boto3.session.Session, region: str) -> List[Dict]:
+    client = boto3_session.client('cloudwatch', region_name=region)
+    response = client.describe_alarms()
+    alarms = []
+    for alarm in response.get('MetricAlarms', []):
+        alarm['region'] = region
+        alarms.append(alarm)
+
+    return alarms
+
+
+def load_cloudwatch_alarm(session: neo4j.Session, alarms: Dict, current_aws_account_id: str, aws_update_tag: int) -> None:
+    session.write_transaction(_load_cloudwatch_alarm_tx, alarms, current_aws_account_id, aws_update_tag)
+
+
+@timeit
+def _load_cloudwatch_alarm_tx(tx: neo4j.Transaction, alarms: Dict, current_aws_account_id: str, aws_update_tag: int) -> None:
+    query: str = """
+    UNWIND {Records} as record
+    MERGE (alarm:AWSCloudWatchAlarm{id: record.AlarmArn})
+    ON CREATE SET alarm.firstseen = timestamp(),
+        alarm.arn = record.AlarmArn
+    SET alarm.lastupdated = {aws_update_tag},
+        alarm.name = record.AlarmName,
+        alarm.region = record.region,
+        alarm.alarm_actions = record.AlarmActions,
+        alarm.namespace = record.Namespace,
+        alarm.period = record.Period,
+        alarm.state_value = record.StateValue,
+        alarm.statistic = record.Statistic,
+        alarm.actions_enabled = record.ActionsEnabled,
+        alarm.metric_name = record.MetricName
+    WITH alarm
+    MATCH (owner:AWSAccount{id: {AWS_ACCOUNT_ID}})
+    MERGE (owner)-[r:RESOURCE]->(alarm)
+    ON CREATE SET r.firstseen = timestamp()
+    SET r.lastupdated = {aws_update_tag}
+    """
+
+    tx.run(
+        query,
+        Records=alarms,
+        AWS_ACCOUNT_ID=current_aws_account_id,
+        aws_update_tag=aws_update_tag,
+    )
+
+
+@timeit
+def cleanup_cloudwatch_alarm(neo4j_session: neo4j.Session, common_job_parameters: Dict) -> None:
+    run_cleanup_job('aws_import_cloudwatch_alarm_cleanup.json', neo4j_session, common_job_parameters)
+
+
+@timeit
+@aws_handle_regions
+def get_cloudwatch_flowlogs(boto3_session: boto3.session.Session, region: str, current_aws_account_id: str) -> List[Dict]:
+    client = boto3_session.client('ec2', region_name=region)
+    response = client.describe_flow_logs()
```

Comment:
we should implement pagination here.

Ref: https://boto3.amazonaws.com/v1/documentation/api/latest/reference/services/ec2.html#EC2.Paginator.DescribeFlowLogs